src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -3,10 +3,10 @@ import React, { useState, useEffect, useRef } from 'react'
 import cn from 'classnames'
 import { useSpring, animated } from 'react-spring'
 
-import styles from './TabSlider.module.css'
-import { TabSliderProps } from './types'
+import styles from './SegmentedControl.module.css'
+import { SegmentedControlProps } from './types'
```

Comment:
Is the plan for us to do a major release for this change? If not one option is to copy tabslider over but still keep tabslider and add deprecation notice.

Reply:
oh yeah im probably misunderstanding, nice